projects/transfer-journey/src/lib/transfer-journey.module.ts

```diff
@@ -13,6 +13,7 @@ import { TransferJourneyComponent } from './transfer-journey.component';
 import { MakeTransferSuccessViewComponent } from './views/make-transfer-success-view.component';
 import { MakeTransferSummaryViewComponent } from './views/make-transfer-summary-view.component';
 import { MakeTransferViewComponent } from './views/make-transfer-view.component';
+import { EntitlementsGuard, EntitlementsModule } from '@backbase/foundation-ang/entitlements';
```

Comment:
Import without actually using in the file?

Reply:
fixed